ansible/group_vars/production/resources.yml

```
@@ -34,14 +34,14 @@ CLIENT_CPU_LIMITS: "1000m"
 CLIENT_CPU_REQUESTS: "100m"
 CLIENT_MEMORY_LIMITS: "1Gi"
 CLIENT_MEMORY_REQUESTS: "1Gi"
-ETHERPA_CPU_LIMITS: "1000m"
-ETHERPA_CPU_REQUESTS: "100m"
-ETHERPA_MEMORY_LIMITS: "1Gi"
-ETHERPA_MEMORY_REQUESTS: "1Gi"
-ETHERPA_NGINX_CPU_LIMITS: "1000m"
-ETHERPA_NGINX_CPU_REQUESTS: "100m"
-ETHERPA_NGINX_MEMORY_LIMITS: "1Gi"
-ETHERPA_NGINX_MEMORY_REQUESTS: "1Gi"
+ETHERPAD_CPU_LIMITS: "1000m"
+ETHERPAD_CPU_REQUESTS: "100m"
+ETHERPAD_MEMORY_LIMITS: "1Gi"
+ETHERPAD_MEMORY_REQUESTS: "1Gi"
+ETHERPAD_NGINX_CPU_LIMITS: "1000m"
+ETHERPAD_NGINX_CPU_REQUESTS: "100m"
+ETHERPAD_NGINX_MEMORY_LIMITS: "1Gi"
+ETHERPAD_NGINX_MEMORY_REQUESTS: "1Gi"
```

Memory and CPU Addaption 
```suggestion
ETHERPAD_CPU_LIMITS: "2000m"
ETHERPAD_CPU_REQUESTS: "100m"
ETHERPAD_MEMORY_LIMITS: "2Gi"
ETHERPAD_MEMORY_REQUESTS: "2Gi"
ETHERPAD_NGINX_CPU_LIMITS: "2000m"
ETHERPAD_NGINX_CPU_REQUESTS: "100m"
ETHERPAD_NGINX_MEMORY_LIMITS: "1Gi"
ETHERPAD_NGINX_MEMORY_REQUESTS: "1Gi"
```